pkgs/by-name/nu/numbat/package.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "numbat";
+  version = "1.6.3";
+
+  src = fetchFromGitHub {
+    owner = "sharkdp";
+    repo = "numbat";
+    rev = "v${version}";
+    hash = "sha256-r6uPe2NL+6r/fKjf0C/5DLdB5YP3SIo8g8EsDxKP/3g=";
+  };
+
+  cargoHash = "sha256-MPqJjCfIwgK8QigWQYfWAYlg9RNMzF4x+0SprS0raKY=";
+
+  buildNoDefaultFeatures = true;
```

Comment:
Could you explain why this is set?

Reply:
To be honest no. 
As far Im aware of numbat has no features to activate anyway. So we could remove it I guess?

Reply:
Sounds good!